package.json

```diff
@@ -51,8 +51,8 @@
     "@babel/core": "^7.12.9",
     "@babel/preset-env": "^7.12.7",
     "babel-loader": "^8.2.2",
-    "identity-style-guide": "^4.0.0",
-    "uswds": "^2.9.0",
+    "identity-style-guide": "^6.2.0",
```

Comment:
This is going to be addressed in https://github.com/18F/identity-site/pull/777 in the main branch but I figure we can keep it in the PR for now so that a) other devs can access the needed new USWDS features and b) we don't have to wait on the main branch PR (which might not get merged till after the new year). We can remove it once we do the big PR into the main branch.